src/pages/auth/register.tsx

```diff
@@ -0,0 +1,182 @@
+                  type={'text'}
+                  required={true}
+                  name={'username'}
+                  errorIcon={usernameInvalid}
+                  placeholder={'Your name'}
+                  icon={'MdPersonOutline'}
+                  errorPadding={'normal'}
+                ></Input>
+              </motion.div>
+              <motion.div
+                initial={{ opacity: 0 }}
+                animate={{ opacity: 1 }}
+                transition={{ delay: 0.3, duration: 0.7 }}
+              >
+                <Input
+                  onChange={emailChange}
+                  errorMessage={`Invalid email or already used.`}
+                  required={true}
+                  type={'email'}
+                  name={'email'}
+                  errorPadding={'small'}
+                  errorIcon={emailInvalid}
+                  placeholder={'Email'}
+                  icon={'MdEmail'}
+                ></Input>
+              </motion.div>
+              <motion.div
+                initial={{ opacity: 0 }}
+                animate={{ opacity: 1 }}
+                transition={{ delay: 0.4, duration: 0.7 }}
+              >
+                <Input
+                  onChange={passwordChange}
+                  errorPadding={'large'}
+                  errorMessage={`Password must have min 5 and max 20 characters with 1 letter, 1 number and 1 special character.`}
+                  errorIcon={passwordInvalid}
+                  type={'password'}
+                  name={'password'}
+                  placeholder={'Password'}
+                  required={true}
+                  icon={'MdLock'}
+                ></Input>
+              </motion.div>
+            </form>
+          </div>
+          <div className="flex flex-col gap-4">
+            <motion.div
+              initial={{ opacity: 0 }}
+              animate={{ opacity: 1 }}
+              transition={{ delay: 0.5, duration: 0.7 }}
+            >
+              <Button
+                onClick={handleSubmit}
+                buttonType="success"
+                fullWidth={true}
+                title={'Sign Up'}
+              ></Button>
+            </motion.div>
+
+            <motion.div
+              initial={{ opacity: 0 }}
+              animate={{ opacity: 1 }}
+              transition={{ delay: 0.6, duration: 0.7 }}
+            >
+              <Button
+                buttonType="pink"
+                fullWidth={true}
+                title={'Sign Up with Google'}
+              ></Button>
+            </motion.div>
+          </div>
+          <div className="flex justify-end pt-6">
+            <motion.div
+              initial={{ opacity: 0 }}
+              animate={{ opacity: 1 }}
+              transition={{ delay: 0.6, duration: 0.7 }}
+            >
+              <LinkButton
+                href={'/auth/login'}
+                target={'_self'}
+                bgColor={'none'}
```

Comment:
Isn't it the default value? Why are we sending 'none'?

Reply:
little mistake. resolved.